proposals/sandbox/k3s.md

```diff
@@ -0,0 +1,471 @@
+1. Wrapped in simple launcher that handles a lot of the complexity of TLS certificates and kubernetes configuration options
+1. Brings its own dependencies such that just a sane kernel and cgroup mounts are needed. k3s packages the following required dependencies:
+    - containerd
+    - Flannel
+    - CoreDNS
+    - CNI
+    - Host utilities (iptables, socat, etc)
+    - Ingress controller (traefik)
+    - Embedded service loadbalancer
+    - Embedded network policy controller
+  
+### Value-add to the CNCF ecosystem
+k3s is bringing Kubernetes to environments and use cases that are not easily accessible to upstream Kubernetes, such as edge and IoT. In doing so, it is promoting the overall growth and adoption of Kubernetes. When coupled with Kubernetes's explosive growth in the cloud and the datacenter, k3s is helping Kubernetes to become the universal compute platform.
+
+### Alignment with other CNCF projects
+***Does the project align and actively collaborate with other CNCF projects?***
+
+k3s is incredibly well aligned with other CNCF projects. It leverages existing CNCF projects such as Kubernetes, CNI, CRI, CSI, containerd, CoreDNS, and Helm. In doing so, its maintainers contribute to these projects by opening issues, submitting PRs, promoting them on social media, and engaging in technical discussions.
+
+***Does the project require any specific versions of projects (or APIs) to interoperate? (e.g. K8s API, CSI, CNI, CRI)?***
+
+k3s generally seeks to support the latest stable versions of the CNCF projects that it leverages. For projects that are themselves closely tied to Kubernetes, such as CNI, CRI, and CSI, k3s supports the same versions that upstream Kubernetes supports.
+
+***Does the project augment or benefit other CNCF projects?***
+
+As stated, k3s benefits Kubernetes by bringing it to environments that are not easily accessible to upstream Kubernetes. In focusing on secondary use cases and environments, k3s also illuminates problems or bugs in the other CNCF projects it leverages. Here are a few examples: [kubernetes#89554](https://github.com/kubernetes/kubernetes/issues/89554), [cri-tools#585](https://github.com/kubernetes-sigs/cri-tools/issues/585), [kubernetes#88376](https://github.com/kubernetes/kubernetes/issues/88376), and [containerd#2991](https://github.com/containerd/containerd/issues/2991).
+
+### Anticipated use cases
+With hundreds of thousands of downloads and a large and growing community of users, k3s's use cases are well proven. These use cases have already been stated: edge, IoT, ARM devices, and software appliances. One major use case that has not yet been addressed is that of fleet management. This is tied directly to the "edge" and IoT use cases, but is worth calling out explicitly. k3s's ease of deployment, operations, and upgrades makes the managing thousands or even millions of small Kubernetes clusters a possibility.
+
+Alignment with SIG Reference Model
```

Comment:
This part is missing?

Reply:
Yes, could use some guidance on how to answer this. To be honest, I started with it blank based on seeing similar in the PARSEC pr https://github.com/cncf/toc/pull/441/files#diff-ad0d260064d69088bf8419b00db477c1R156